### ISSUE TITLE
[REQUEST] Add command to delete local and remote branches at the same time

### ISSUE THREAD

**Issue by curtisgibby** [akonwi/git-plus#789]
My last Git GUI client, [SmartGit](https://www.syntevo.com/smartgit/), included this as a checkbox option when deleting a local repo.

![delete local and remote branches](https://user-images.githubusercontent.com/1086964/55254102-a5754a00-521c-11e9-9479-174a7fefaaee.jpg)

I think the following would work, added right after the existing `delete-local-branch` and `delete-remote-branch` in [gitplus.js](https://github.com/akonwi/git-plus/blob/7efa9b0e8b5fab8133e70bea56f2ce383b069f5e/lib/git-plus.js#L154-L157)

```javascript
"git-plus:delete-local-and-remote-branch": () => {
  git.getRepo().then(repo => GitDeleteBranch(repo))
    .then(repo => GitDeleteBranch(repo, { remote: true }));
},
```

I understand that this could potentially be done with a [Custom Command](https://github.com/akonwi/git-plus/tree/7efa9b0e8b5fab8133e70bea56f2ce383b069f5e#experimental-features), but I think that there's enough value there that it would worthwhile to include in the full repo.

**Comment by akonwi** [comment]
Yep this is a great idea. Could you open up a PR and add the command to `lib/git-plus-commands.coffee` too? Also, how about having the label be "Delete Branch (Local and Remote)"?

**Comment by curtisgibby** [comment]
I like that idea. However, as far as I understand, the label is derived from the "slug" (or whatever you'd call it). How would we be able to generate the parenthesis in the label from something like `delete-branch-local-and-remote`?

**Comment by akonwi** [comment]
The [`CommandRegistry::add`](https://atom.io/docs/api/v1.36.0/CommandRegistry#instance-add) method has a couple signatures. What you'll do is similar to [the Fetch All command](https://github.com/akonwi/git-plus/blob/master/lib/git-plus.js#L210)

**Comment by curtisgibby** [comment]
PR #794 opened.
